.github/workflows/gradle.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+env:
+  ghidra-url: https://ghidra-sre.org/ghidra_9.1.2_PUBLIC_20200212.zip
+  ghidra-zip-filename: ghidra_9.1.2_PUBLIC.zip
+  ghidra-directory: ghidra_9.1.2_PUBLIC
+
+on:
+  push:
+    branches: [ master]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Download Ghidra release
+      uses: carlosperate/download-file-action@v1.0.3
+      id: download-ghidra
+      with:
+        file-url: ${{ env.ghidra-url }}
+        file-name: ${{ env.ghidra-zip-filename }}
+        location: nsis/dist/
+    
+    - name: Unzip Ghidra
+      uses: TonyBogdanov/zip@1.0
+      with:
+        args: unzip -qq ${{ steps.download-ghidra.outputs.file-path }} -d .
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '11.0.2'
+
+    - name: Build extension
+      uses: eskatos/gradle-command-action@v1
+      with:
+        gradle-version: '5.0'
+        build-root-directory: ${{ github.workspace }}/nsis
+        arguments: '-PGHIDRA_INSTALL_DIR=${{ github.workspace }}/${{ env.ghidra-directory }}'
+
+
+    - name: Run tests
+      uses: eskatos/gradle-command-action@v1
+      with:
+        gradle-version: '5.0'
+        build-root-directory: ${{ github.workspace }}/nsis
+        arguments: 'clean test -PGHIDRA_INSTALL_DIR=${{ github.workspace }}/${{ env.ghidra-directory }}'
```

Comment:
I don't think you need `clean` here. Once you remove it you can also remove the `-PGHIDRA_INSTALL_DIR...` part. So the arguments will just be `arguments: test`

Unless there is a reason to delete everything that was just built, rebuilt it, and then run the tests.

Reply:
Good point, I can remove  the `clean`, but not the -PGHIDRA_INSTALL_DIR. I don't know why but it still tries to build the project when I just use `test` as an argument, so I get an error because I didn't specify -PGHIGRA_INSTALL_DIR

Reply:
Got it, I have the environment variable `GHIGRA_INSTALL_DIR` set on my box, so that is why it worked for me.

Reply:
I didn't find a way to specify the build directory for the tests so I left the -PGHIDRA_INSTALL_DIR in the gradle.yml file.